Samples/iOS-Swift/iOS-Swift/AppDelegate.swift

```diff
@@ -162,3 +163,19 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // swiftlint:enable force_cast
     }
 }
+
+private extension AppDelegate {
+    // previously tried putting this in an AppDelegate.load override in ObjC, but it wouldn't run until after a launch profiler would have an opportunity to run, since SentryProfiler.load would always run first due to being dynamically linked in a framework module. it is sufficient to do it  before calling SentrySDK.startWIthOptions to clear state for testProfiledAppLaunches because we don't make any assertions on a launch profile the first launch of the app in that test 
+    func maybeWipeData() {
```

Comment:
`l`: We could rename this to wipeAppData and call it above when `ProcessInfo.processInfo.arguments.contains("--io.sentry.wipe-data")` is true. The name maybeWipeData is a bit confusing to me.